internal/operators/lvm/lvm_operator.go

```diff
@@ -147,56 +171,113 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
+	}
+
+	// blocking auto assign for LVM
+	if role == models.HostRoleAutoAssign {
+		err := fmt.Errorf("Cannot Get host Requirment for auto assign role for host %s", host.ID)
+		log.WithError(err)
+		return nil, err
+	}
+
+	// default to Master
 	return preflightRequirements.Requirements.Master.Quantitative, nil
 }
 
 // GetPreflightRequirements returns operator hardware requirements that can be determined with cluster data only
 func (o *operator) GetPreflightRequirements(context context.Context, cluster *common.Cluster) (*models.OperatorHardwareRequirements, error) {
+	// unsupportted before 4.11
+	if ok, _ := common.BaseVersionLessThan(o.Config.LvmMinOpenshiftVersion, cluster.OpenshiftVersion); ok {
+		return &models.OperatorHardwareRequirements{}, fmt.Errorf("LVM is not supported")
+	}
+
 	dependecies, err := o.GetDependencies(cluster)
 	if err != nil {
 		return &models.OperatorHardwareRequirements{}, err
 	}
-	return &models.OperatorHardwareRequirements{
+
+	// baseHardwareRequirement, _ := GetBaseRequirements(GetPage)
+
+	requirements := models.OperatorHardwareRequirements{
 		OperatorName: o.GetName(),
 		Dependencies: dependecies,
 		Requirements: &models.HostTypeHardwareRequirementsWrapper{
 			Master: &models.HostTypeHardwareRequirements{
+				Qualitative: []string{},
 				Quantitative: &models.ClusterHostRequirementsDetails{
-					CPUCores: o.config.LvmCPUPerHost,
-					RAMMib:   o.getLvmMemoryPerHostMib(context, cluster),
-				},
-				Qualitative: []string{
-					"At least 1 non-installation disk with no partitions or filesystems",
+					CPUCores: 4,
+					RAMMib:   16384,
 				},
 			},
 			Worker: &models.HostTypeHardwareRequirements{
-				Quantitative: &models.ClusterHostRequirementsDetails{},
+				Qualitative: []string{},
+				Quantitative: &models.ClusterHostRequirementsDetails{
+					CPUCores: 2,
+					RAMMib:   8192,
+				},
 			},
 		},
-	}, nil
-}
+	}
+	snoBaseRequirements := &models.ClusterHostRequirementsDetails{
+		CPUCores: 8,
+		RAMMib:   16384,
+	}
 
-func (o *operator) GetFeatureSupportID() models.FeatureSupportLevelID {
-	return models.FeatureSupportLevelIDLVM
-}
+	qualitativeRequirements := []string{
+		"At least 1 non-boot disk on one or more host",
+		fmt.Sprintf("%d GiB of additional RAM", o.Config.LvmMemoryPerHostMiB),
+		fmt.Sprintf("%d additional CPUs for each non-boot disk", o.Config.LvmCPUPerHost),
+	}
 
-func (o *operator) getLvmMemoryPerHostMib(ctx context.Context, cluster *common.Cluster) int64 {
-	log := logutil.FromContext(ctx, o.log)
-	greaterOrEqual, err := common.BaseVersionGreaterOrEqual(LvmsMinOpenshiftVersionForNewResourceRequirements, cluster.OpenshiftVersion)
-	if err != nil {
-		log.Warnf("Error parsing cluster.OpenshiftVersion: %s, setting lvms memory requirement to %d", err.Error(), o.config.LvmMemoryPerHostMiB)
-		return o.config.LvmMemoryPerHostMiB
+	// unsupportted multi node before 4.15
+	if ok, _ := common.BaseVersionLessThan(openshiftVersion4_15, cluster.OpenshiftVersion); ok {
```

Comment:
maybe this constant can be `lvmOnMultinodeMaxVersion` or something along the lines?

Reply:
dont like the term `maxVersion`  we can use `lvmOnMultinodeMinVersion` which is 4.15
but I think the version here is more readable than something like min/max